AutoGear.lua

```diff
@@ -143,10 +152,11 @@ function AutoGearInitializeDB(defaults, reset)
 	end
 end
 
--- We run the IsClassic and IsTBC check before function definition to prevent poorer performance
-if (IsClassic or IsTBC or IsWotLK) then
+---Specializations appeared only in Mists Of Pandaria. We also have make changes to Cataclysm with prefered talent tree
+---later
+if TOC_VERSION_CURRENT < 50000 then
```

Comment:
Please use a variable in this comparison and all similar comparisons instead of hard-coding TOC numbers in-line.

Reply:
fixed